src/test/resources/db/fill/issue-entity/V013__issue-entity-fill.sql

```diff
@@ -0,0 +1,74 @@
+CREATE OR REPLACE FUNCTION items_init()
```

Comment:
let's write a little documentation in each fill script what kind of data it generates